style.css

```diff
@@ -193,6 +193,76 @@
     row-gap: 15px;
   }
 
+  .detail-container {
+    display: none;
+    position: fixed;
+    inset: 0;
+    background-color: rgba(0, 0, 0, 0.5);
+  }
```

Comment:
- [ ] Please adjust the size of the mobile popup, it shouldn't be full height if the content isn't filling the whole screen, please check this example of what happened:

![image](https://user-images.githubusercontent.com/20567503/195155364-539344ef-9e22-4b57-aead-2b617d691f7a.png)

Kindly fix this issue so that your portfolio appears professional.